sdk/textanalytics/ai-text-analytics/src/analyzeSentimentResult.ts

```diff
@@ -188,21 +188,22 @@ function convertGeneratedSentenceSentiment(
     confidenceScores: sentence.confidenceScores,
     sentiment: sentence.sentiment,
     text: sentence.text,
-    minedOpinions: sentence.aspects?.map(
-      (aspect: SentenceAspect): MinedOpinion => ({
-        aspect: {
-          confidenceScores: {
-            ...(aspect.confidenceScores as SentimentConfidenceScores),
-            neutral: 0
+    minedOpinions: sentence.aspects ?
+      sentence.aspects.map(
+        (aspect: SentenceAspect): MinedOpinion => ({
+          aspect: {
+            confidenceScores: {
+              ...(aspect.confidenceScores as SentimentConfidenceScores),
+              neutral: 0
```

Comment:
Didn't we also resolve to refine this type as well? Since `neutral` is always 0, I think we should just omit it.

Reply:
done.